src/config/webpack.config.js

```diff
@@ -10,32 +10,14 @@ const here = p => path.join(__dirname, p);
 const babelWhitelist = require("./babel-whitelist.json");
 
 const config = {
-	entry: [
-		here("setAssetPath.js"),
-		"url-polyfill",
-		"core-js",
-		"whatwg-fetch",
-		path.resolve("./src/index.js"),
-	],
+	entry: [here("setAssetPath.js"), "url-polyfill", "core-js", "whatwg-fetch", path.resolve("./src/index.js")],
 	output: {
 		chunkFilename: "[id].[chunkhash].js",
 		filename: "bundle.js",
 		path: path.resolve(process.cwd(), "dist"),
 	},
 	resolve: {
-		alias: {
```

Comment:
Did you really mean to remove that ? Unit tests of orc-shared are failing, you will need to figure that out. Look for Travis build reports.